src/hooks/useNodePagination.js

```diff
@@ -0,0 +1,103 @@
+import appConfig from 'app.config';
+import { client } from 'client';
+import { useRef } from 'react';
+
+/**
+ * Default prepass items for posts. This lists all the pieces of data we need
+ * for each post node. Running the following through `prepass` ensures that
+ * all of the data is there when we need it, and no cascading requests happen.
+ *
+ * @see https://gqty.dev/docs/client/helper-functions#prepass
+ */
+export const defaultPostPrepassItems = [
```

Comment:
Great work. The risk here is that developers will forget that they have to update the list of prepass items if they add another property. GQty makes it easier to fall into that trap. Let's see what the others thing in that case.